### ISSUE TITLE
Upgrade to recent SQLite version

### ISSUE THREAD

**Issue by david-bouyssie** [ryanhomer/sqlite3-compression-encryption-vfs#2]
Is there any chance to get this great extension available for a more recent version of SQLite?

Thanks!

**Comment by techrah** [comment]
@david-bouyssie I don't have time at the moment but in the meantime, can you describe the issues you are having with more recent versions of SQLite? Are you getting specific errors, for example? Do you have any ideas on what would need to be done? Are you willing to attempt to do this and submit a pull request?

**Comment by david-bouyssie** [comment]
I have not tested yet.
I was just referring to this known issue: https://github.com/ryanhomer/sqlite3-compression-encryption-vfs#knows-issues
Is this issue already solved?
Regarding the PR, I don't think I will be able to manage it, but I can do some testing for sure.
Thanks!

**Comment by techrah** [comment]
I just spent the last few days tracking down the incompatibility issue (see #5). The fix now makes it compatible up to version 3.15.2. Unfortunately, the next version, 3.16.0, introduces some breaking API changes so I'll have to address this separately.

Also, in my testing with versions 3.12.0 and 3.15.2, I had to increase the `block_size` of the lower pager to `8192` in order to get it to work properly. See updated `README` on how to do this. I'm not yet sure why this was necessary.
